app/src/main/res/values-fr/strings.xml

```diff
@@ -3,38 +3,53 @@
     <string name="digiview_feed">DigiView\nfeed</string>
     <string name="waiting_for_usb_device">Attente de périphérique USB…</string>
     <string name="usb_device_approved">Périphérique USB approuvé</string>
-    <string name="usb_device_detached_waiting">Périphérique USB détaché\n\nAttente de périphérique USB…</string>
```

Comment:
lets remove these modifications as you made a separate PR for them ^^

Reply:
I removed all french translations updates in this branch